src/app/src/components/Graph/HoverOverlay.tsx

```diff
@@ -116,8 +116,27 @@ const HoverOverlay = (props: Props): React.ReactElement => {
 
   const getTooltipText = React.useCallback((): string => {
     const revision = buildRevisionFromX(offsetX - Offset.LEFT);
-    return `Revision: ${formatSha(revision.value)}, Artifact: ${hoveredArtifact}`;
-  }, [buildRevisionFromX, hoveredArtifact, offsetX]);
+
+    const dataSeries = data.find(d => d.key === hoveredArtifact);
+    const buildDate = dataSeries[revision.index].data.timestamp;
+    const formattedDate =
+      // Output date as YYYY-MM-DD
+      buildDate
+        .toLocaleDateString('ja', {
```

Comment:
I don't have experience with this– is this instructing to use a Japanese formatted date?

There's another location that's using `toLocaleString` instead of `toLocaleDateString`:
https://github.com/paularmstrong/build-tracker/blob/fe104e7df668da6197b0d5aed9010b1e382d3b02/src/app/src/components/BuildInfo.tsx#L66

I wonder if we could standardize them. What do you think? Also, `date-fns` are pulled in here and there if you'd prefer.

Reply:
realized i never replied here!

My idea was showing the datetime in `<YYYY-MM-DD> <time_as_your_locale_prefers>`.  Basically forcing YYYY-MM-DD because programming.  But time is 24hr vs AM/PM depending on the user.  

Hypothetically with the right options one could get this format out of a single call to .toLocaleString() (which would be formatting as datetime)... but I'm kinda skeptical that's possible.

and yes i've used this hack of leveraging the japanese locale to get YYYY-MM-DD before. :)  it's... a solution. :)  

These .toLocale*String() methods are supported great in the browser, and handle most of what folks use date libs for. (in node, support is a little more behind (but decent), however often support depends on [if ICU was part of the build](https://nodejs.org/dist/latest-v14.x/docs/api/intl.html#intl_options_for_building_node_js).)